### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by soroshsabz**
Add cmake instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -732,6 +732,26 @@ For more details please check the project [wiki](https://github.com/fnc12/sqlite
...
+git clone https://github.com/fnc12/sqlite_orm.git sqlite_orm
+cd sqlite_orm
+mkdir compile
+cd compile
+cmake ..
+cmake --build .
+sudo make install
+```
+
+then you can just include `sqlite_orm.h` that install in system-wide header files location or if you use cmake build system you can just add below commands in CMakeLists.txt
```
Comment (fnc12) [inline README.md]: 1) `that is installed` instead of `that install`
2) `in case you use cmake build` instead of `if you use cmake build`

**Comment by fnc12** [comment]
this is a great PR. Thanks for contributing. Please apply fixes I wrote about in comments and I'm ready to merge it

File: README.md (hunk 1 above)
Comment (soroshsabz) [reply README.md]: @fnc12 I changed it

**Comment by soroshsabz** [comment]
@fnc12 Thanks for reviewing, I resolve your review :)

**Pull request merged**